app/Http/Controllers/SmartXpScreenController.php

```diff
@@ -33,23 +33,19 @@ public function protopenersTimetable()
      * @param $stop
      * @return object[]
      */
-    public function bus($stop)
+    public function bus(Request $request)
     {
         try {
-            $departures = json_decode(stripslashes(file_get_contents("https://api.9292.nl/0.1/locations/enschede/$stop/departure-times?lang=en-GB")));
-
-            return $departures->tabs[0]->departures;
+            return file_get_contents("http://v0.ovapi.nl/tpc/$request->tpc_id,$request->tpc_id_other");
         } catch (Exception $e) {
-            return [(object) [
-                'time' => '00:00',
-                'service' => '',
-                'mode' => (object) [
-                    'name' => 'Error in API!',
-                ],
-                'realtimeText' => '',
-                'realtimeState' => '9292',
-                'destinationName' => 'who knows?',
-            ]];
+            return json_encode((object) [
+                'dragon'=>(object) [
```

Comment:
Please make this a useful error

Reply:
made it a normal json response where the error is moved to the blade file